bin/checkver.ps1

```diff
@@ -213,7 +213,14 @@ while ($in_progress -gt 0) {
     }
 
     if ($jsonpath) {
-        $ver = json_path $page $jsonpath
+        # Return only a single value if regex is absent
+        $single = [String]::IsNullOrEmpty($regex)
+        # If reverse is ON and regex is ON,
+        # Then reverse would have no effect because regex handles reverse
+        # on its own
+        # So in this case we have to disable reverse
+        $reverse = $reverse -and $single
```

Comment:
@niheaven this change won't rewrite the global reverse variable right?

I am not that well versed with PowerShell's shenanigans but to me it looks like it would

Reply:
😄 Yes, my mistake. It change $reverse's value in global scope. See recent review, and I advise changing `$single` to `$noregex`